modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -143,6 +143,13 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro assertPredefinedDateValuePresent {
+		AssertElementPresent(
+			key_fieldName = "${fieldName}",
```

Comment:
@LucasAcacio, actually, shouldn't it be named as `fieldLabel` instead?

After all, **Date** and **Copy of Date** are the label of the fields, right?

Reply:
Rebased and fixed! Done!

Reply:
@lucasperj, can we rename `key_fieldName` to `key_fieldLabel` to keep a naming consistency?

Please consider the other macros using the path `FormViewBuilder#PREDEFINED_VALUE_AT_FORM_DATE_FIELD` because they'll need an update too. If you verify that it's too much effort, please let me know and we discuss. Feel free to reach me out through Slack. :wink: